plugins/outputs/prometheus_client/sample.conf

```
@@ -40,3 +40,9 @@
 
   ## Export metric collection time.
   # export_timestamp = false
+
+  # # Add one or more type mappings to force a metric type for a specific field
+  # # NOTE: This is only supported by metric_verison = 2
```

```suggestion
  ## Add one or more type mappings to force a metric type for a specific field
  ## NOTE: This is only supported by metric_verison = 2
```

Since not all types are supported, I would add which ones can be used.